Marlin/Marlin_main.cpp

```diff
@@ -12832,17 +12808,22 @@ void setup() {
   #endif
 
   MYSERIAL.begin(BAUDRATE);
+  while(!MYSERIAL);
```

Comment:
Could you change this to a simple delay? Thank you.

[Reference at Teensy forums](https://forum.pjrc.com/threads/26015-3D-Printer-Software-with-Teensy-3-1?p=145275&viewfull=1#post145275)